streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -104,16 +104,18 @@ public void process(Exchange exchange) {
 			return;
 		}
 		
-		if (parquetUtil.getParquetPath() != null) {
+		if (!parquetUtil.getParquetPath().isEmpty()) {
```

Comment:
Did you remember to document this feature in the readme file? For example, it would be good to have a section that describes how to switch between the output/sink modes and/or both?

Reply:
> Did you remember to document this feature in the readme file? For example, it would be good to have a section that describes how to switch between the output/sink modes and/or both?

Never mind, I see you have already done that!